### PR TITLE
adding shivam mamgain to team list

### PR DESCRIPTION
**Pull request by Hasala123**
I completely misunderstood the task at first. Still learning to use github I hope this is correct

### CODE REVIEW

File: team/index.html [hunk 1]
```diff
@@ -2473,6 +2473,30 @@ <h1>Alumni</h1>
...
+						<span class="speaker-name">Shivam Mamgain</span>
+						<span>Programmer</span>
+					</div>
+				</div>
+
+
+
+
+
+
```
Comment (nikit19) [inline team/index.html]: Avoid these blank lines

**Review by nikit19 — CHANGES_REQUESTED** [review]
Can you add it to the end of the list.

File: team/index.html [hunk 2]
```diff
@@ -2472,7 +2472,6 @@ <h1>Alumni</h1>
 						<span>Full Stack Developer</span>
 					</div>
 				</div>
-
```
Comment (kushthedude) [inline team/index.html:2475]: Revert this change

Comment (Hasala123) [reply team/index.html:2475]: I made those changes because nikit19 told me to remove the unnecessary blank line i had left. Can you be more specific?

**Review by mariobehling — APPROVED** [review]
